docs/src/guides/drupal9/redis.md

```diff
@@ -1,149 +1,172 @@
 ---
 title: "Using Redis with Drupal 9.x"
 sidebarTitle: "Redis"
-description: |
-    Add Redis caching to your existing Drupal site.
+description: Add Redis caching to your existing Drupal site.
 weight: -70
 ---
 
-If you are using the Platform.sh-provided Drupal template, most of this work is already done for you.
-Redis is already configured and is enabled after the installation is complete.
+Redis is a fast open-source in-memory database and cache, 
+useful for application-level caching. 
+For more information on this service, see [our dedicated Redis page](../../add-services/redis.md) 
+or the [official Redis documentation](https://redis.io/docs/).
 
-Note that this `redis` service is ephemeral, meaning it doesn't persist if the container moves or is shut down.
-Your app must treat it as ephemeral and not rely on it being there.
+The following steps let you:
 
-There are two possible solutions:
+- Add and configure Redis for Drupal 9.x if you have deployed Drupal manually.
+- Fine-tune your existing configuration if you have deployed Drupal 9 using a [Platform.sh template](../../development/templates.md).
 
-* Change the service to persistent Redis (`redis-persistent`).
-* Clean the cache each time your app starts in the `start` key in [your web configuration](../../create-apps/app-reference.md#web-commands).
+## Before you begin
 
-If you are working from an older repository or migrating a pre-built site to Platform.sh, see the instructions below.
+Make sure that:
 
-## Requirements
+- You have [deployed Drupal 9.x on Platform.sh](../drupal9/deploy/_index.md).
+- You have [installed the Platform.sh CLI](../../administration/cli/).
+- You have [installed Composer](https://getcomposer.org/).
+  Composer is the tool recommended by both Platform.sh and Drupal 
+  to [add the Redis module to your Drupal project](../../guides/drupal9/redis.md#3-add-the-drupal-module). 
+  Platform.sh also recommends you use Composer to manage your whole site and its dependencies.
+- You have [installed the Platform.sh Config Reader library](../../guides/drupal9/deploy/customize.md#install-the-config-reader).
 
-### Add a Redis service
+Note that, by default, Redis is an ephemeral service.
+This means that the Redis storage isn't persistent 
+and that data can be lost when a container is moved, shut down 
+or when the service hits its memory limit.
+
+To solve this, you can change the [service type](../../add-services/redis.md#service-types) 
+to persistent Redis (`redis-persistent`).
+Alternatively, you can clean the cache each time your app starts 
+via the `start` key in [your web configuration](../../create-apps/app-reference.md#web-commands).
+
+## Add a Redis service
 
 {{% endpoint-description type="redis" noApp=true onlyLanguage="php" php=true /%}}
```

Comment:
❓ Was I right in thinking the source content should be reworked when the main Redis page itself is reworked as a whole? I considered it out of the scope of this PR for now and kept it as is.